server/config.py

```diff
@@ -59,8 +59,8 @@
 TWILIO_TOKEN = os.getenv("TWILIO_TOKEN", "")
 TWILIO_TTL = os.getenv("TWILIO_TTL", 86400)
 
-COTURN_HOSTS = os.getenv('COTURN_HOSTS', "").split(',')
-COTURN_KEYS = os.getenv('COTURN_KEYS', "").split(',')
+COTURN_HOSTS = os.getenv('COTURN_HOSTS', "geosearchef.de").split(',')
+COTURN_KEYS = os.getenv('COTURN_KEYS', "wop0CC7EUxrrcavH7aKCvaUjmpT3ZwW7N5ozYI7Jps").split(',')
```

Comment:
yes, I've seen what I did there, this shouldn't have been published, the file needs to be reverted